src/thread_event.c

```diff
@@ -267,28 +306,30 @@ thread_event_trigger_batch_update(tsd_t *tsd, uint64_t accumbytes,
 }
 
 void
-thread_event_trigger(tsd_t *tsd, bool delay_event) {
+thread_event_trigger(tsd_t *tsd, event_ctx_t *ctx, bool is_alloc,
+    bool delay_event) {
```

Comment:
Is `is_alloc` always equal to `ctx->is_alloc`?

Reply:
Yep good point. Will update.